app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if not command:
+        return
+    command_list = command.split()
+
+    if len(command_list) == 3:
+        mv, file_name, new_file_name = command_list
+        if mv == "mv":
+            if "/" in new_file_name:
+                path_file = new_file_name.split("/")
```

Comment:
your app will be cross-platform and will use either / or \ depending on OS. Read checklist. Fix it related to this case